### PR TITLE
e2e: Use pod instead of ds for logcompress file toucher

### PR DESCRIPTION
**Pull request by mrogers950**
Also only modify and check on a single node, to help speed things up.

### CODE REVIEW

File: tests/e2e/helpers.go [hunk 1]
```diff
@@ -1147,27 +1160,6 @@ func editFileOnNodes(f *framework.Framework, namespace string) error {
 	return nil
 }
 
-func addALottaFilesOnNodes(f *framework.Framework, namespace string) error {
```
Comment (rhmdnd) [inline tests/e2e/helpers.go:1150]: IIUC - we were testing compression by adding a bunch of files to the host. Now we're testing compression another way?

Comment (mrogers950) [reply tests/e2e/helpers.go:1150]: Nope, it's the same method. I just converted what was a daemonset into a pod on a single node.

File: tests/e2e/helpers.go [hunk 2]
```diff
@@ -467,13 +473,20 @@ func cleanAideAddedFilesDaemonset(namespace string) *appsv1.DaemonSet {
...
-// 20000 might not be enough
-// lowered to 10000
-func addLotsOfFilesDaemonset(namespace string) *appsv1.DaemonSet {
-	return privCommandDaemonset(namespace, "aide-add-lots-of-files",
-		"for i in `seq 1 10000`; do mktemp \"/hostroot/etc/addedbytest$i.XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX\"; done || true",
-	)
+// This pod is to stuff the aide log with added files, to force compression. The trick is in the long temp name that pads the aide log.
+// 10000 files.
+func addCompressionTestFilesOnNode(t *testing.T, f *framework.Framework, ctx *framework.Context, node, namespace string) error {
+	pod := privPodOnNode(namespace, "test-compression-pod", node, compressionFileCmd)
```
Comment (rhmdnd) [inline tests/e2e/helpers.go:479]: Does the pod name need to be unique, or will it be run serially?

Comment (rhmdnd) [inline tests/e2e/helpers.go:479]: Ok - the `compressionFileCmd` is what I was missing before. I kept looking for what created the files, but now I see that it's the command.

**Review by rhmdnd — COMMENTED** [review]
Looks good - just one more question inline.

File: tests/e2e/helpers.go (hunk 2 above)
Comment (mrogers950) [reply tests/e2e/helpers.go:479]: nah, it's serial

**Comment by mrogers950** [comment]
Madhu verified this fixed his issues with this test on Z

**Pull request merged**